packages/docusaurus-theme-classic/src/translations.ts

```diff
@@ -54,9 +67,17 @@ function translateNavbar(
   if (!navbarTranslations) {
     return navbar;
   }
+
+  const {logo} = navbar;
+
+  if (logo) {
+    logo.alt = navbarTranslations[`logo.alt`]?.message ?? navbar.logo?.alt;
+  }
```

Comment:
I'd prefer to not mutate the logo object here but instead create a copy with the changes needed

It's likely that I'll want to make these translation methods pure in the future and may ensure that by freezing the input objects to ensure they are not modified.

Will do the fix and merge

Reply:
Is it necessary for me to do something?

Reply:
No I did the changes already ;)